demo-shell-ng2/config/webpack.common.js

```diff
@@ -145,21 +145,17 @@ module.exports = {
         inline: true,
         proxy: {
             '/ecm': {
-                target: {
-                    host: "0.0.0.0",
-                    protocol: 'http:',
-                    port: 8080
-                },
+                target:  "http://adfdev.lab.alfresco.me",
```

Comment:
do we now always link to external server from the demo shell?

Reply:
I removed it